C++/Navio/Navio2/RCOutput_Navio2.cpp

```diff
@@ -0,0 +1,26 @@
+#include "RCOutput_Navio2.h"
+
+RCOutput_Navio2::RCOutput_Navio2()
+{
+
+}
+
+bool RCOutput_Navio2::initialize()
+{
+    if (!pwm.init(PWN_OUTPUT)) {
+        fprintf(stderr, "Output Enable not set. Are you root?\n");
+        return 0;
+    }
+
+    pwm.enable(PWN_OUTPUT);
+}
+
+void RCOutput_Navio2::setFrequency(float frequency)
+{
+    pwm.set_period(PWN_OUTPUT, frequency);
```

Comment:
Why only on zeroth channel? It doesn't make sense! Please.

Reply:
Yes, you are right, I will correct all mistakes.